From b63e1e68d8f989ff1b62cd8e6acc5547592e38a2 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 12 Sep 2018 20:55:41 +0800
Subject: [PATCH] net: wireless: rockchip_wlan: ssv6xxx: Fix gcc warning

drivers/net/wireless/rockchip_wlan/ssv6xxx/ssvdevice/ssv_cmd.c: In function 'ssv_cmd_dump':
drivers/net/wireless/rockchip_wlan/ssv6xxx/ssvdevice/ssv_cmd.c:387:3: error: argument 1 null where non-null expected [-Werror=nonnull]
   strcpy(ssv6xxx_result_buf, "ssv6xxx_result_buf = NULL!!\n");
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Change-Id: I883d9f17814c25c0d8f7611dc99944f6b072b002
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/net/wireless/rockchip_wlan/ssv6xxx/ssvdevice/ssv_cmd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/rockchip_wlan/ssv6xxx/ssvdevice/ssv_cmd.c b/drivers/net/wireless/rockchip_wlan/ssv6xxx/ssvdevice/ssv_cmd.c
index 033e7a09b40e..4687afe70b76 100644
--- a/drivers/net/wireless/rockchip_wlan/ssv6xxx/ssvdevice/ssv_cmd.c
+++ b/drivers/net/wireless/rockchip_wlan/ssv6xxx/ssvdevice/ssv_cmd.c
@@ -384,7 +384,7 @@ static int ssv_cmd_dump(int argc, char *argv[])
     int s;
  if(!ssv6xxx_result_buf)
  {
-  strcpy(ssv6xxx_result_buf, "ssv6xxx_result_buf = NULL!!\n");
+  printk("ssv6xxx_result_buf = NULL!!\n");
   return -1;
  }
     if (argc != 2)
-- 
2.35.3

